site/en/tutorials/load_data/text.ipynb

```diff
@@ -352,22 +352,13 @@
         "### Prepare the dataset for training"
       ]
     },
-    {
-      "cell_type": "markdown",
-      "metadata": {
-        "id": "HSDO9JZYrK-O"
-      },
-      "source": [
-        "Note: The Preprocessing APIs used in this section are experimental in TensorFlow 2.3 and subject to change."
-      ]
-    },
     {
       "cell_type": "markdown",
       "metadata": {
         "id": "N6fRti45Rlj8"
       },
       "source": [
-        "Next, you will standardize, tokenize, and vectorize the data using the `preprocessing.TextVectorization` layer.\n",
+        "Next, you will standardize, tokenize, and vectorize the data using the `layers.TextVectorization` layer.\n",
```

Comment:
Small suggestion:

```suggestion
        "Next, you will standardize, tokenize, and vectorize the data using the `tf.keras.layers.TextVectorization` layer.\n",
```

This way, the API docs should be auto-linked when the notebook is viewed on tensorflow.org.

LMKWYT

Reply:
The rule for the auto-links is "at least one `.` in the quoted path". The link will work either way.

Reply:
OK!

Reply:
@MarkDaoust , @8bitmp3  - Should i need to do any thing from my end for the recommend suggestion above? Please let me know. Thanks!